Docker build fails

The following command fails at the  current master  `sudo docker build -t electrs-app .
`  on Ubuntu 20.04.1 LTS x86_64.

The output is:
```
  Compiling electrs v0.8.5 (/home/user)                                                                                                                                                                     
   Compiling rocksdb v0.12.4                                                                                                                                                                                 
error: linking with `cc` failed: exit code: 1                                                                                                                                                                
  |                                                                                                                                                                                                          
  = note: "cc" "-Wl,--as-needed" "-Wl,-z,noexecstack" "-m64" "-L" "/usr/local/rustup/toolchains/1.41.1-x86_64-unknown-linux-gnu/lib/rustlib/x86_64-unknown-linux-gnu/lib" "/home/user/target/release/deps/electrs-6e2ee0ac9a1f4942.electrs.3lnq89eb-cgu.15.rcgu.o" "-o" "/home/user/target/release/deps/electrs-6e2ee0ac9a1f4942" "-Wl,--gc-sections" "-pie" "-Wl,-zrelro" "-Wl,-znow" "-Wl,-O1" "-nodefaultlibs" "-L" "/home/user/target/release/deps" "-L" "/home/user/target/release/build/secp256k1-sys-86d90aac8f530ec1/out" "-L" "/home/user/target/release/build/librocksdb-sys-e1757150eca7f7b5/out" "-L" "/home/user/target/release/build/librocksdb-sys-e1757150eca7f7b5/out" "-L" "/ho

.... <lots and lots of errors>.....



 /usr/bin/ld: plain_table_builder.cc:(.text._ZN7rocksdb17PlainTableBuilder6FinishEv+0x575): undefined reference to `rocksdb::Footer::EncodeTo(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >*) const'                                                                                                                                                                           
          collect2: error: ld returned 1 exit status                                                                                                                                                         
          

error: aborting due to previous error

error: failed to compile `electrs v0.8.5 (/home/user)`, intermediate artifacts can be found at `/home/user/target`

Caused by:
  could not compile `electrs`.

To learn more, run the command again with --verbose.
The command '/bin/sh -c cargo install --path .' returned a non-zero code: 101
```

--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -28,10 +28,10 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: build
-          args: --release --all
+          args: --locked --release --all
 
       - name: Test
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --release --all
+          args: --locked --release --all

--- a/Dockerfile
+++ b/Dockerfile
@@ -14,7 +14,7 @@ RUN chown -R user .
 
 USER user
 
-RUN cargo install --path .
+RUN cargo install --locked --path .
 
 # Electrum RPC
 EXPOSE 50001

